spec/models/tslr_claim_spec.rb

```diff
@@ -242,7 +223,8 @@
     end
 
     it "validates the claim is not ineligible" do
```

Comment:
This could also do with rewording, I think. Maybe "validates the claim's eligibility"?

Reply:
👍🏽